torch/csrc/jit/passes/python_print.cpp

```diff
@@ -776,6 +795,15 @@ struct PythonPrintPass {
           printValueList(stmt, node->inputs(), "[", "]");
         }
       } break;
+      case prim::DictConstruct: {
+        stmt << "annotate(" << node->output()->type()->python_str() << ", ";
```

Comment:
Minor nit: for lists we do not annotate if it is not necessary. For instance if the list is non-empty, or if it is empty but the type is `List[Tensor]`. Is something like that possible here? It will make the outputted form more readable and potentially more robust to use changing things around.